src/wp-includes/style-engine/class-wp-style-engine.php

```diff
@@ -26,26 +26,21 @@
 #[AllowDynamicProperties]
 final class WP_Style_Engine {
 	/**
-	 * Style definitions that contain the instructions to
-	 * parse/output valid Gutenberg styles from a block's attributes.
-	 *
-	 * For every style definition, the following properties are valid:
-	 *
-	 *  - classnames    => (array) An array of classnames to be returned for block styles.
-	 *                     The key is a classname or pattern.
-	 *                     A value of `true` means the classname should be applied always.
-	 *                     Otherwise, a valid CSS property (string) to match the incoming value,
-	 *                     e.g. "color" to match var:preset|color|somePresetSlug.
-	 *  - css_vars      => (array) An array of key value pairs used to generate CSS var values.
-	 *                     The key is a CSS var pattern, whose `$slug` fragment will be replaced with a preset slug.
-	 *                     The value should be a valid CSS property (string) to match the incoming value,
-	 *                     e.g. "color" to match var:preset|color|somePresetSlug.
-	 *  - property_keys => (array) An array of keys whose values represent a valid CSS property,
-	 *                     e.g. "margin" or "border".
-	 *  - path          => (array) A path that accesses the corresponding style value in the block style object.
-	 *  - value_func    => (string) The name of a function to generate a CSS definition array
-	 *                     for a particular style object. The output of this function should be
-	 *                     `array( "$property" => "$value", ... )`.
+	 * Style definitions that contain the instructions to parse/output valid Gutenberg styles from a block's attributes.
+	 *
+	 * For every style definition, the follow properties are valid:
```

Comment:
I'm not a native English speaker but "the following properties" seems to be the correct (or at least more consistent) form here.

Reply:
You're right. Thanks for picking up on that. I blindly copied and pasted.

Reply:
Looks like it's been committed. I can create a new PR since it's an embarrassing spelling error (for me).

Changeset: https://github.com/WordPress/wordpress-develop/pull/5256